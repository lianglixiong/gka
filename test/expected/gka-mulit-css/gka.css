.gka-wrap {
    width: 400px;
    height: 300px;
}

.animation {
    width: 400px;
    height: 300px;
    background-image: url("./img/a_a01.png");
    background-size: 400px 300px;
    background-repeat: no-repeat, no-repeat;
    animation-name: keyframes-a, keyframes-b;
    animation-duration: 0.36s, 0.44s;
    animation-delay: 0s, 0.36s;
    animation-iteration-count: 1, infinite;
    animation-fill-mode: forwards, forwards;
    animation-timing-function: steps(1), steps(1);
}

@keyframes keyframes-a {
    0% {
        width: 400px;
        height: 300px;
        background-image: url("./img/a_a01.png");
        background-size: 400px 300px;
    }
    11.11% {
        background-image: url("./img/a_a02.png");
    }
    22.22% {
        background-image: url("./img/a_a03.png");
    }
    33.33% {
        background-image: url("./img/a_a04.png");
    }
    44.44% {
        background-image: url("./img/a_a05.png");
    }
    55.56% {
        background-image: url("./img/a_a06.png");
    }
    66.67% {
        background-image: url("./img/a_a07.png");
    }
    77.78% {
        background-image: url("./img/a_a08.png");
    }
    88.89%,
    100% {
        background-image: url("./img/a_a09.png");
    }
}

@keyframes keyframes-b {
    0% {
        width: 400px;
        height: 300px;
        background-image: url("./img/b_a10.png");
        background-size: 400px 300px;
    }
    9.09% {
        background-image: url("./img/b_a11.png");
    }
    18.18% {
        background-image: url("./img/b_a12.png");
    }
    27.27% {
        background-image: url("./img/b_a13.png");
    }
    36.36% {
        background-image: url("./img/b_a14.png");
    }
    45.45% {
        background-image: url("./img/b_a15.png");
    }
    54.55% {
        background-image: url("./img/b_a16.png");
    }
    63.64% {
        background-image: url("./img/b_a17.png");
    }
    72.73% {
        background-image: url("./img/b_a18.png");
    }
    81.82% {
        background-image: url("./img/b_a19.png");
    }
    90.91%,
    100% {
        background-image: url("./img/b_a20.png");
    }
}