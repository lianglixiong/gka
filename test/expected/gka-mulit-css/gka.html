<html>
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0,maximum-scale=1">
    
    <title>gka-preview</title>
<link href="./gka.css" rel="stylesheet" type="text/css">
</head>
<body>

    <div class="gka-wrap">
        <div id="gka"></div>
    </div>
    <script>
    function preloadImage(names, cb, prefix){
        window.gkaCache = window.gkaCache || [];
        var n = 0,img,imgs = {};
        names.forEach(function(name) {
            img = new Image();
            window.gkaCache.push(img);
            img.onload = (function(name, img) {
                return function() {
                    imgs[name] = img;
                    (++n === names.length) && cb && cb(imgs);
                }
            })(name, img);
            img.src = (prefix || '') + name;
        });
    }

    preloadImage(["a_a01.png","a_a02.png","a_a03.png","a_a04.png","a_a05.png","a_a06.png","a_a07.png","a_a08.png","a_a09.png","b_a10.png","b_a11.png","b_a12.png","b_a13.png","b_a14.png","b_a15.png","b_a16.png","b_a17.png","b_a18.png","b_a19.png","b_a20.png"], function() {
        document.getElementById('gka').className += " animation"
    }, "img/")
    </script>
</body>
</html>